samples/azure/asb-and-asbs-side-by-side/sample.md

```diff
@@ -0,0 +1,24 @@
+---
+title: ASBS to ASB Side By Side
+reviewed: 2017-10-17
+component: ASBS
+related:
+- transports/azure-service-bus
+- nservicebus/bridge
+---
+
+
+TBD...
+
+NOTE:
```

Comment:
Can this be turned into a snippet rather than hard-coding in markdown?
Would it work with csproj file given it's an xml file?
Ref: https://github.com/Particular/docs.particular.net#defining-snippets

Reply:
@Particular/docs-maintainers any way to include a `csproj` xml file in snippets scanning?

Reply:
I've raised an issue: https://github.com/Particular/docs.particular.net/issues/4022